### PR TITLE
documentation(EJ2MVC-490): Updated UG content for rating in Asp.net Core and MVC.

### PR DESCRIPTION
**Pull request by VigneshwaranGovindharajan**
### Feature description

Need to add UG Documentation for rating control in Asp.net Core and MVC platform.

1. Getting started
2. Precision modes
3. Labels
4. Tooltip
5. Selection
6. Templates
7. Events
8. Appearance
9. Accessibility

### Solution description

Added UG Documentation for rating control in Asp.net Core and MVC platform.

### Output screenshots

NA

### Areas affected and ensured

None (New Component)

### Test cases

NA

### Test bed sample location

NA

### Additional checklist

* Did you run the automation against your implementation? NA
* Is there any API name change? NA.
* Is there any existing behavior change of other features due to this code change? No
* Does your new code introduce new warnings or binding errors? No
* Does your code pass all FxCop and StyleCop rules? Yes
* Did you record this case in the unit test or UI test? Yes

### CODE REVIEW

File: ej2-asp-core-mvc/rating/EJ2_ASP.MVC/appearance.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+description: Learn here all about Appearance in Syncfusion ##Platform_Name## Rating control of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: Appearance
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+# Appearance in Asp.Net MVC Rating Control
+
+You can also customize the appearance of rating component.
```
Comment (NavinVinayagam) [inline ej2-asp-core-mvc/rating/EJ2_ASP.MVC/appearance.md:13]: rating control

File: ej2-asp-core-mvc/rating/EJ2_ASP.MVC/appearance.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+{% include code-snippet/rating/appearance/items-count/razor %}
+{% endhighlight %}
+{% highlight c# tabtitle="ItemsCount.cs" %}
+{% include code-snippet/rating/appearance/items-count/itemscount.cs %}
+{% endhighlight %}
+{% endtabs %}
+
+{% endif %}
+
+![Asp.Net MVC Rating Component with Items count](images/rating-items-count.png)
```
Comment (NavinVinayagam) [inline ej2-asp-core-mvc/rating/EJ2_ASP.MVC/appearance.md:32]: Change rating component to rating control in all places

File: ej2-asp-core-toc.html [hunk 3]
```diff
@@ -1849,7 +1849,14 @@
 		<li>Rating
 			<ul>
 				<li><a href="/ej2-asp-core/rating/getting-started">Getting Started</a></li>
-				<li><a href="/cr/aspnetcore-js2/Syncfusion.EJ2.Inputs.Rating.html">API Reference</a></li>
```
Comment (NavinVinayagam) [inline ej2-asp-core-toc.html:1852]: Why you have removed the API reference link?

File: ej2-asp-mvc-toc.html [hunk 4]
```diff
@@ -1818,7 +1818,14 @@
 		<li>Rating
 			<ul>
 				<li><a href="/ej2-asp-mvc/rating/getting-started">Getting Started</a></li>
-				<li><a href="/cr/aspnetmvc-js2/Syncfusion.EJ2.Inputs.Rating.html">API Reference</a></li>
```
Comment (NavinVinayagam) [inline ej2-asp-mvc-toc.html:1821]: Why you have removed the API reference link?

**Review by NavinVinayagam — CHANGES_REQUESTED** [review]

File: ej2-asp-core-mvc/rating/EJ2_ASP.MVC/appearance.md (hunk 1 above)
Comment (VigneshwaranGovindharajan) [reply ej2-asp-core-mvc/rating/EJ2_ASP.MVC/appearance.md]: updated as per comments.

File: ej2-asp-core-mvc/rating/EJ2_ASP.MVC/appearance.md (hunk 2 above)
Comment (VigneshwaranGovindharajan) [reply ej2-asp-core-mvc/rating/EJ2_ASP.MVC/appearance.md]: updated as per comments.

File: ej2-asp-core-toc.html (hunk 3 above)
Comment (VigneshwaranGovindharajan) [reply ej2-asp-core-toc.html:1852]: It mistakenly removed when I resolve conflict, now I added API reference link.

File: ej2-asp-mvc-toc.html (hunk 4 above)
Comment (VigneshwaranGovindharajan) [reply ej2-asp-mvc-toc.html:1821]: It mistakenly removed when I resolve conflict, now I added API reference link.

**Review by NavinVinayagam — CHANGES_REQUESTED** [review]

File: ej2-asp-core-mvc/rating/EJ2_ASP.MVC/events.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+{% include code-snippet/rating/events/valuechanged-event/razor %}
+{% endhighlight %}
+{% highlight c# tabtitle="ValueChanged.cs" %}
+{% include code-snippet/speeddial/events/valuechanged-event/valuechanged.cs %}
+{% endhighlight %}
+{% endtabs %}
+
+## OnItemHover
+
+The rating component triggers the [OnItemHover](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.Inputs.Rating.html#Syncfusion_EJ2_Inputs_Rating_OnItemHover) passed as an event argument provides the details of the hovered item.
```
[the remark this answers is not in the document]
Comment (VigneshwaranGovindharajan) [reply ej2-asp-core-mvc/rating/EJ2_ASP.MVC/events.md]: updated as per comments.

File: ej2-asp-core-mvc/rating/EJ2_ASP.MVC/events.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+documentation: ug
+---
+
+# Events in Rating Control
+
+This section describes the rating events that will be triggered when appropriate actions are performed. The following events are available in the rating component.
+
+## BeforeItemRender
+
+The rating component triggers the [BeforeItemRender](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.Inputs.Rating.html#Syncfusion_EJ2_Inputs_Rating_BeforeItemRender) event before rendering each rating item.
```
[the remark this answers is not in the document]
Comment (VigneshwaranGovindharajan) [reply ej2-asp-core-mvc/rating/EJ2_ASP.MVC/events.md]: updated as per comments.

File: ej2-asp-core-mvc/rating/EJ2_ASP.MVC/tooltip.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+{% endif %}
+
+![Asp.Net MVC Rating Component with Tooltip](./images/rating-tooltip.png)
+
+## Tooltip template
+
+You can use the [TooltipTemplate](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.Inputs.Rating.html#Syncfusion_EJ2_Inputs_Rating_TooltipTemplate) tag directive to specify a custom template for the `Tooltip` of the rating. The current value of the rating will be passed as the `value` when building the content of the tooltip. This allows you to include dynamic information about the rating in the template.
```
[the remark this answers is not in the document]
Comment (VigneshwaranGovindharajan) [reply ej2-asp-core-mvc/rating/EJ2_ASP.MVC/tooltip.md]: updated as per comments.

File: ej2-asp-core-mvc/rating/EJ2_ASP.MVC/labels.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+{% endtabs %}
+
+{% endif %}
+
+![Asp.Net MVC Rating Component with different label positions](./images/rating-label-positions.png)
+
+
+## Label template
+
+You can use the [LabelTemplate](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.Inputs.Rating.html#Syncfusion_EJ2_Inputs_Rating_LabelTemplate) tag directive to specify a custom template for the `Label` of the rating. The current value of the rating will be passed as the `value` when building the content of the label. This allows you to include dynamic information about the rating in the template.
```
[the remark this answers is not in the document]
Comment (VigneshwaranGovindharajan) [reply ej2-asp-core-mvc/rating/EJ2_ASP.MVC/labels.md]: updated as per comments.

**Review by NavinVinayagam — APPROVED** [review]

**Pull request merged**